package.json

```diff
@@ -14,8 +14,8 @@
     "node": ">=16.14.0"
   },
   "scripts": {
-    "start": "lerna run start --parallel --ignore @okta/odyssey-ie-11-sandbox",
-    "start:ie11": "lerna run start --scope @okta/odyssey-ie-11-sandbox",
+    "start": "yarn workspace @okta/odyssey-storybook start",
+    "start:ie11": "yarn workspace @okta/odyssey-ie-11-sandbox start",
```

Comment:
@jeffbelser-okta  have you tested if this loads fine too ?

Reply:
yup!